### PR TITLE
New headings loop function

### PR DESCRIPTION
**Pull request by maciejpi**
@incuna/frontend please review

### CODE REVIEW

**Comment by wytrych** [comment]
Looks alright

File: incuna-sass/functions/_headings.sass [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// sass function to loop through H tags
+@function headings($from:1, $to:6)
+    @if $from == $to
+        @return 'h#{$from}'
+    @else
+        @return 'h#{$from},' + headings($from+1, $to)
+
+// Call it passing 2 numeric arguments. For example,
```
Comment (hippogriffic) [inline incuna-sass/functions/_headings.sass]: I think @grahamgilchrist would say the example needs to go above the mixin

**Comment by maciejpi** [comment]
the example moved to the top

**Comment by hippogriffic** [comment]
nice :+1:

**Pull request merged**